### ISSUE TITLE
[Effect Suggestion] Blimp Bros

### ISSUE THREAD

**Issue by davidman2** [gta-chaos-mod/ChaosModV#3485]
**Name of effect**
Potential name(s) for the effect

**Duration**
- [ ] Long
- [ ] Short
- [x] No duration

**Effect Description**
Description of the effect; what should it do?
It will put everyone including player into a blimp.

**Comment by ReiDaTecnologia** [comment]
I don't think it is a good idea as it is a big ass vehicle, it would mostly like explode immediately and cause a ton of lag because of it

**Comment by MoneyWasted** [comment]
Closing as similar to the existing `PlayerBlimpStrats` effect — putting everyone into blimps is a variation of the same concept.
